### PR TITLE
Added correct definition for createIndex method of IDBObjectStore

### PR DESCRIPTION
**Pull request by aksharpatel47**
Fixes Microsoft/Typescript#5832

### CODE REVIEW

**Comment by mhegazy** [comment]
:+1:

File: inputfiles/overridingTypes.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
  [
+     {
+         "kind": "method",
+         "interface": "IDBObjectStore",
+         "name": "createIndex",
+         "signatures": ["createIndex(name: string, keyPath: string | string[], optionalParameters?: any): IDBIndex"]
+     },
+	 {
```
Comment (zhengbli) [inline inputfiles/overridingTypes.json]: niche: the indentation seems inconsistent. Use spaces instead of tabs.

**Comment by zhengbli** [comment]
Other than the comment, looks good to me :+1:. Thanks for contributing @aksharpatel47 !

**Comment by aksharpatel47** [comment]
@zhengbli There are a lot of other IndexedDB definitions missing such as multiEntry property of an Index, interfaces for options when creating an object store and an index. I can add them, but should I create a new issue and a pull request or add them here?

**Comment by zhengbli** [comment]
@aksharpatel47 An issue should be created, the missing types can all belong to a single issue to avoid too much efforts. Some of them may be corrected next time we take an updated XML spec files from the Edge team, but it would be great if we can get a quick fix before that (which might be a painful process with many breaking changes). Thanks!

**Pull request merged**